requirements.txt

```diff
@@ -0,0 +1,5 @@
+Cython==3.0.10
+bitstruct==8.19.0
+pandas==2.2.2
+numpy==1.26.4
+hdf5==1.10.6
```

Comment:
I think that `hdf5` is not a direct dependency, just having `pandas` in the list should be enough IMHO.
Moreover there is a missing dependency that is `bpack`.
Finally fo you mind to rename the file into `requirements-dev.txt`?

Reply:
mmmm, I will prefer putting all the dependencies in the pyproject.toml to avoid redundancy.
Even though some user may prefer the requirements.txt approach.

- Installing pandas per se does not set hdf5 unfortunately. 
- Correct, bpack was missing there.

I am thinking to update the pyproject.toml and readme to include this if you agree

Reply:
> mmmm, I will prefer putting all the dependencies in the pyproject.toml to avoid redundancy.

It is OK for me, and if you use the "editable" pip installation it should work also for developers that need to modify the source code and quickly run it.

The "requirements.txt" file could be handy for developers that prefer to work on the code without pip-installing it (but in the end such people should be able to create a working environment looking at the `pyproject.toml` file.

> Installing pandas per se does not set hdf5 unfortunately.

OK, the right way to go then should be to set a dependency on `pandas[hdf5]` in the `pyproject.toml`.

Reply:
Many thanks! I am gonna update the toml with your revision